### ISSUE TITLE
Doesn't work on bootstrap modal

### ISSUE THREAD

**Issue by rajeem** [gasparesganga/jquery-loading-overlay#11]
jquery loading overlay doesn't work when applied on objects inside bootstrap modal

**Comment by gasparesganga** [comment]
Please provide some details:
- is it being created in the DOM?
- is there a problem with the placement or with the z-index?

Can you provide the full code to reproduce the behaviour?

**Comment by rajeem** [comment]
hi @gasparesganga,

please take a look at this sample code https://jsfiddle.net/5z7thes5/

**Comment by gasparesganga** [comment]
A jsFiddle is worth a million words! There are two simple fixes to your code:

1. You are missing a point before the `.modal-body` class in the selector
2. Bootstrap uses a `z-index` so you need to set an higher one for LoadingOverlay.

```javascript
$("#myModal").on("shown.bs.modal", function(){
    $("#myModal .modal-body").LoadingOverlay("show", {
        zIndex : 9999
    });
});
```

If you don't want to specify a `zIndex` option in every LoadingOverlay call, you can take advantage of the **`$.LoadingOverlaySetup`** method, initializing it only once at the beginning of your code.

**Comment by gasparesganga** [comment]
By the way, in brand new [v1.5.2](https://github.com/gasparesganga/jquery-loading-overlay/commit/9e0fb111c7d0605634e9f4d5654bb17006492093)  `zIndex` option now defaults to 9999.
Cheers

**Comment by mdrahiem** [comment]
First of all thanks for this great plugin. it decreases lot of my work. I want to implement this loader into materialize modal. can you please suggest me fix for this. 

https://codepen.io/anon/pen/aYBQzE

**Comment by gasparesganga** [comment]
Why are you using an obsolete version of LoadingOverlay hosted on jqueryscript?
Please use current version, if you want to use a CDN there's a link in the [Get It](https://gasparesganga.com/labs/jquery-loading-overlay/#get-it) section of the official page.

Secondly, this issue is about another topic and has been closed since december 2016!!! Please open a new one.

Now, about your question, it is surely possible, but what are you trying to achieve? It's not clear at all looking at your code. Please describe it better (**in a new issue**, don't forget it).

**Comment by mdrahiem** [comment]
Sorry for writing on this issue. I will create new one right now. Thanks much for your reply.
